### ISSUE TITLE
Create Release and Finish Quality Gate

### ISSUE THREAD

**Issue by tom-rm-meyer-ISST** [eclipse-tractusx/puris#328]
**As** PURIS developer,
**I want** to create the release for the R24.05 version,
**so that** I an adopter may use it accordingly.

blocks: #321 

## Hints / Details

- Release must be created prior to Release Check

### Outcome
- ...
### Acceptance Criteria
- Helm Chart Release has been created
- Foss Release has been created (docker hub, github)
- Changelog has been written
- Known Knowns have been documented in README

## Out of Scope

- ...
